### DISCUSSION TITLE
Player-made missiles/self-destruct/AOE-damage

### DISCUSSION THREAD

**Discussion by josh-hemphill** [tsunamayo/Starship-EVO#4628] [Suggestions]
I think any kind of bomb or missiles could be a lot of fun to give some purpose to "flack-like" guns created from the wide spread of the gatling. I'm not sure what it would look like in this game, but having some kind of AOE bomb block that would let us create custom missiles with different ranges and yields would be really fun. The way Space Engineers did it, I think is probably not the way to go, since in that game, custom missiles were almost always too complicated and ineffective to be worth it. Even if the missiles aren't super effective, if they're a lot easier to implement, then it would make a lot of sense to send volleys of missiles to try and overwhelm flack/close-range countermeasures.

**Comment by Malus-Draco** [comment]
The simplest solution to this would be a fire and forget unguided missile for now, just give us a block that explodes when given a signal and size/number of block determines yield the same was reactors determine power and fuel by block. with that I could make a rocked that explodes when its close to target, what would be nice is if we could send a signal to fire engines/thrusters... then we could have semi-guided system.
